hosting/src/store/index.ts

```diff
@@ -15,7 +16,13 @@ const store = createStore<PTVuexState>({
     currentUser: null,
     events: []
```

Comment:
I know you didn't write this, but I was thinking this morning that this should be `futureEvents` not `events`, and we should update the query.

Reply:
If we don't load past events, both `/events` & `/event/:slug` (for past events) need to run their own queries.

I'm starting to think that because we're very events centric, we might actually consider loading ALL events.

Reply:
DONE - I left it as events (but fixed some types & constants)